lib/Gitlab/Api/Projects.php

```diff
@@ -496,11 +496,12 @@ public function removeLabel($project_id, $name)
 
     /**
      * @param int $project_id
+     * @param array $params
      * @return mixed
      */
-    public function fork($project_id)
+    public function fork($project_id, array $params = array())
     {
-        return $this->post('projects/'.$this->encodePath($project_id).'/fork');
+        return $this->post('projects/'.$this->encodePath($project_id).'/fork', $params);
```

Comment:
Could you add `$params` validation using `OptionResolver`?

Reply:
I don't know how to use;

Reply:
Here is an example of how to use `OptionResolver` for a `post` request: https://github.com/m4tthumphrey/php-gitlab-api/blob/032b7cca7b376686522d00bcb404bc9167a6e0c4/lib/Gitlab/Api/RepositoryFiles.php#L35-L68.